### PR TITLE
Tmp/form fix reg exp

### LINKED ISSUE #62: Form field validations
- [x] Add red asterisk to fields that are required by customer so they know it has to be filled out .. i.e maybe phone can be left as not required since email has been provided

- [x] Form feedbacks if customer hit submit without filling out required fields or consider disabling submit button until required fields are filled out

- [x] Check that domain.com is added to every email passed perhaps using regex to check

- [x] Check that phone number is at least/most 10digits and maybe append a dash after each one

### LINKED ISSUE #63: Set timeout for contact confirmation modals
[ ] Add settimeout to error and success modal after email is sent

### PR DESCRIPTION
**Pull request by MrGDS2**
required

### CODE REVIEW

File: madgiraffestudios/src/components/Booking/Booking.js [hunk 1]
```diff
@@ -80,39 +81,42 @@ const [message, setMessage] = useState ('');
...
-                     <Input type="email" name="email" value={email} onChange={e=>setEmail(e.target.value)} placeholder="Email@gmail.com" />
+                 <FormGroup className="form-group required">
+                     <Label for="email" className="d-block text-left control-label">Email</Label>
+                     <Input type="email" name="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" value={email} onChange={e=>setEmail(e.target.value)} placeholder="Email@gmail.com" title="Please put in a valid email address" required />
                  </FormGroup>
-                 <FormGroup>
+                 <FormGroup >
                      <Label for="phone" className="d-block text-left">Phone</Label>
-                     <Input type="tel" name="phone" value={phone} onChange={e=>setPhone(e.target.value)} placeholder="(917)-000-0000" />
+                     <Input type="tel" name="phone" pattern="\d{3}[\-]\d{3}[\-]\d{4}" value={phone} onChange={e=>setPhone(e.target.value)} placeholder="917-000-0000" title="Please put in phone number format 917-XXX-XXX"/>
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Booking/Booking.js]: standard placeholder for phones is 123-456-7890, please consider updating is per user experience and eliminating confusion

File: madgiraffestudios/src/components/Booking/Booking.scss [hunk 2]
```diff
@@ -28,3 +28,7 @@
 .swal-button--confirm:hover{
     background-color:  #9381FF !important;
 }
+
+.form-group.required .control-label:after {
+    content:"*";color:red;
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Booking/Booking.scss:34]: these should be on a new line

File: madgiraffestudios/src/components/Booking/Booking.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ const [message, setMessage] = useState ('');
       name: fullName,
       phone: phone, 
       from_name: email,
-      to_name: 'Madgiraffestudios@gmail.com',
+      to_name: 'MadGiraffeSTuDios',
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Booking/Booking.js]: how would they know the email domain

**Review by estherkoko — CHANGES_REQUESTED** [review]

File: madgiraffestudios/src/components/Booking/Booking.js [hunk 4]
```diff
@@ -53,7 +53,8 @@ const [message, setMessage] = useState ('');
         title: "Message on the way!",
         text: `We can't wait to work with you, ${fullName}!`,
         icon: "success",
-        dangerMode: false
+        timer: 3000
```
Comment (estherkoko) [inline madgiraffestudios/src/components/Booking/Booking.js]: - [x] This should be added to the failure message as well and perhaps set the timeout for 5000 for the error to give the client enough time to read the information and grab the email

**Review by estherkoko — CHANGES_REQUESTED** [review]

File: madgiraffestudios/src/components/Booking/Booking.js (hunk 4 above)
Comment (estherkoko) [reply madgiraffestudios/src/components/Booking/Booking.js]: - [x] can you pass this as a parameter after timer -- to get rid of the OK buttons
`buttons: {
          cancel: null
        }`

File: madgiraffestudios/src/components/Booking/Booking.js (hunk 1 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/Booking/Booking.js]: will address in next PR

File: madgiraffestudios/src/components/Booking/Booking.js (hunk 3 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/Booking/Booking.js]: this change was for kae to view not the client it's for the email template

Comment (MrGDS2) [reply madgiraffestudios/src/components/Booking/Booking.js]: ooh i see your right

Comment (MrGDS2) [reply madgiraffestudios/src/components/Booking/Booking.js]: Addressed

File: madgiraffestudios/src/components/Booking/Booking.js (hunk 1 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/Booking/Booking.js]: changed in next PR

**Comment by MrGDS2** [comment]
All changes addressed

**Review by estherkoko — APPROVED** [review]

**Pull request merged**